corehq/toggles.py

```diff
@@ -923,6 +923,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
     help_link='https://confluence.dimagi.com/display/ccinternal/Support+Flag',
 )
 
+CASE_PROPERTY_HISTORY = StaticToggle(
+    'case_property_history',
+    'Shows a modal on the case property page allowing you to see the history of various case properties',
+    TAG_SOLUTIONS,
```

Comment:
Err. Actually can you add the namespace domain here?

Reply:
Also I think if this is going to get released, it shouldn't be solutions? @sheelio would know more

Reply:
@proteusvacuum 
Ya, I would product path it if we do intend to release it.

Reply:
Updated